ui/common/Table.tsx

```diff
@@ -18,8 +18,12 @@ export default function Table({
               key={index}
               className={clsx({
                 'w-full': index === 0,
-                'w-1/3': index > 0 && index < headings.length - 1,
-                'w-1/6': index === headings.length - 1,
+                'w-1/2': headings.length === 2, // only two columns in the table
```

Comment:
I added a check for the number of columns so that when there are only 2 columns, they each take up 50% of the table width. Not doing this would force this table to adopt the column widths of the table in the chapter 7 mempool-transaction-1 exercise, which has 4 columns and those settings look weird on this new chapter 4 table.